nss-day-cohort-45/tabloidfullstack-gibbering-gizmos
# Cwd post

## Changes Made:
1. Added the add post methods and functions/components for front and back end functionality.

## Steps to test:
1. Click the New Post section of the navbar.
2. Fill out the form for creating a post and click "Save"
3. Make sure that when saving the post the user is taken to the details for the new post.
